pkg/kubelet/dockershim/docker_service.go

```diff
@@ -536,6 +549,20 @@ func (ds *dockerService) getDockerVersionFromCache() (*dockertypes.Version, erro
 	return dv, nil
 }
 
+func (ds *dockerService) getDockerInfoFromCache() (*dockertypes.Info, error) {
+	// We only store on key in the cache.
+	const dummyKey = "info"
```

Comment:
nit: maybe rename dummyKey to something slightly more descriptive?
infoCacheKey?

Reply:
was mimicking https://github.com/kubernetes/kubernetes/pull/104287/files#diff-5aa5f12c1a4791a1ab2b9a0378b27c2a11d5b8cf7f24c26adec1a8b3caf4354dR540